jupyterhub/jupyterhub/base/jupyterhub-singleuser-profiles-configmap.yaml

```diff
@@ -54,3 +54,16 @@ data:
               spark_image: 'quay.io/radanalyticsio/openshift-spark-py36:2.4.5-2'
             return:
               SPARK_CLUSTER: 'metadata.name'
+      gpuTypes:
+      - type: gpu_one
```

Comment:
Should we just remove this one now? Iirc we added it in very much as a placeholder

Reply:
I guess there's no harm with leaving it in, and in a way this shows how to specify multiple gpu types